async-nats/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 0.13.0
+## Added
+* Add Auth - username/password & token by @Jarema in https://github.com/nats-io/nats.rs/pull/408
+* Support sending and receiving messages with headers by @caspervonb in https://github.com/nats-io/nats.rs/pull/402 
+* Add async server errors callbacks by @Jarema in https://github.com/nats-io/nats.rs/pull/397
+
+## Changed
+* Wait between reconnection attempts by @caspervonb in https://github.com/nats-io/nats.rs/pull/407
+* Limit connection attempts by @caspervonb in https://github.com/nats-io/nats.rs/pull/400
+
+## Other
+* Remove redundant doctests by @Jarema in https://github.com/nats-io/nats.rs/pull/412
```

Comment:
This is just CI, no need to mention?

Reply:
Just wanted to be transparent and mention that the change in CI runs is not loosening CI requirements for CI to pass on PR, but just removal of redundancy.